seqr/management/commands/copy_project_tags.py

```diff
@@ -0,0 +1,30 @@
+from django.core.management.base import BaseCommand, CommandError
```

Comment:
CommandError isn;t used

Reply:
have removed the unused imports in latest commits